### PR TITLE
preening

### PR DESCRIPTION
**Pull request by galenwp**
Still working on this, but can PR again

### CODE REVIEW

**Comment by matildepark** [comment]
You're going to want to pull @vvisigoth's copyedits into this.

**Comment by galenwp** [comment]
i'll merge and push

**Comment by matildepark** [comment]
Pull again. Are you done with this?

**Comment by galenwp** [comment]
Ok I think this should be ready.

File: content/understanding-urbit/project-history.md [hunk 1]
```diff
@@ -22,12 +22,16 @@ Urbit OS isn’t just an interface — it’s a complete system. You should also
 
 ![](https:///media.urbit.org/site/understanding-urbit/project-history/history-4.svg)
 
-At the beginning, Urbit was just a few people with the right combination of imagination and discipline to try to rebuild computing. Now it's a community of thousands of address space holders, dozens of open source developers, at least two independent companies ([Tlon](https://tlon.io) — which works mostly on kernel development and interface design, and [urbit.live](https://urbit.live) — which is building services on the platform), and who knows how many fans. 
+At the beginning, Urbit was just a few people with the right combination of imagination and discipline to try to rebuild computing. Now it's a community of thousands of address space holders, dozens of open source developers, at least two independent companies ([Tlon](https://tlon.io) — which works mostly on kernel development and interface design, and [urbit.live](https://urbit.live) — which is building services on the platform), and who knows how many fans.
 
+<<<<<<< HEAD
+![](https://media.urbit.org/site/understanding-urbit/project-history/project-history-05.svg)
+=======
```
Comment (joemfb) [inline content/understanding-urbit/project-history.md:29]: This pr added conflict markers.

**Comment by matildepark** [comment]
Oh, damn. You’re right. Will fix.

**Pull request merged**